pennylane/devices/tests/test_properties.py

```diff
@@ -233,8 +224,11 @@ def test_returns_state(self, device_kwargs):
         if "returns_state" not in cap:
             pytest.skip("No returns_state capability specified by device.")
 
-        qnode = qml.QNode(qfunc_no_input, dev)
-        qnode()
+        @qml.qnode(dev)
+        def circuit():
+            return qml.state()
```

Comment:
How come we can have an empty circuit with state? If the reason is that the IonQ plugin doesn't support `qml.state`, then it may still be good to add an operation, such that when it does start supporting `qml.state`, then the test doesn't fail.